### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by irvifa**
Added link for rust-lang and developer forum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # The Rust Programming Language
 
-Rust is a fast systems programming language that guarantees
+[(Rust)] is a fast systems programming language that guarantees
```
Comment (huonw) [inline README.md]: I think you only need [] here, not the ().

**Comment by Gankro** [comment]
@rvu95 can you fit the only nit?

Also cc @brson I know you've been conservative about this sort of thing in the past.

**Comment by brson** [comment]
I'm already pretty dissatisfied with the existing first paragraph here and don't feel like the addition makes makes it more beautiful as written.

cc @steveklabnik

**Comment by steveklabnik** [comment]
I am also pretty ambivalent about those changes.

**Comment by steveklabnik** [comment]
@rvu95 it's been a loooong time since we've heard from you, and this PR hasn't been mergeable for two weeks. If you're interested in fixing up this PR, let me know, but until then, I'm going to give it a close.

**Pull request closed without merging**